### ISSUE TITLE
Project name on Project components is h3 should be h2 or h1

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1210]
The Project name on the /project/projectName view in the dashboard is an h3 and comes before the  sub-component titles which are h1 in the structure of the document. The Project name should be changed to h1 and the sub-component/page titles should be h3.

**Comment by rmmayo** [comment]
PR: #1439

**Comment by rmmayo** [comment]
The rule states "Usually the best practice is to start the main content of a web page with a level 1 heading (h1), with no other headings before this high-level heading" - all pages use the SubPageHeader component, and despite its name it is the main content area.

https://dequeuniversity.com/rules/axe/4.1/page-has-heading-one

**Comment by mosefer** [comment]
:+1:
